js/main.js

```diff
@@ -17,10 +23,115 @@ function storeValues(event) {
   data.entries.unshift(fieldEntries);
 
   data.nextEntryId++;
-
+  var fieldList = renderEntry(fieldEntries);
+  hiddenDiv.prepend(fieldList);
   placeHolder.setAttribute('src', './images/placeholder-image-square.jpg');
+  viewSwap('entries');
+  if (data.entries) {
```

Comment:
This conditional isn't necessary. If an entry has just been submitted, you don't need to check if there is anything in the array, because you just saved an entry, so it is guaranteed that there are now entries.